terraform/modules/diego/diego_elb_main.tf

```diff
@@ -0,0 +1,56 @@
+/*
+ * Variables required:
+ *   stack_description
+ *   elb_subnets
+ *   
+ */
+
+
+resource "aws_security_group" "diego-elb-sg" {
+  name = "diego-elb-sg"
```

Comment:
Hard coding this name will cause an issue once you run from production stack. May want to use `stack_description` in here to make it unique.